src/components/Footer/Footer.js

```diff
@@ -1,94 +1,111 @@
-import React from 'react'
-import styled from 'styled-components'
-import { GiMedicines} from 'react-icons/gi'
-import { GrFacebookOption } from 'react-icons/gr'
-import { AiOutlineInstagram } from 'react-icons/ai'
-import { RiLinkedinFill } from 'react-icons/ri'
-import { CgTwitter } from 'react-icons/cg'
-
-
+import React from "react";
+import styled from "styled-components";
+import { GiMedicines } from "react-icons/gi";
+import { GrFacebookOption } from "react-icons/gr";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { RiLinkedinFill } from "react-icons/ri";
+import { CgTwitter } from "react-icons/cg";
 
 const MainFooter = styled.div`
-position: absolute;
-width: 100%;
-bottom: 0;
-`
-const Container = styled.div`
-`
+  position: absolute;
```

Comment:
the cleaner approach would be creating some common stylesheets and define these as css. After you hand out such a code to your collegues, there will be no react programmer who would be happy about seeing styling variables inside the react code. I know it works this way, but it's not the cleanest way to do it :)